system/modules/isotope/dca/tl_page.php

```diff
@@ -23,12 +23,14 @@
 /**
  * Extend tl_page palettes
  */
-\Haste\Dca\PaletteManipulator::create()
-    ->addLegend('isotope_legend', 'publish_legen', \Haste\Dca\PaletteManipulator::POSITION_BEFORE)
-    ->addField('iso_config', 'isotope_legend', \Haste\Dca\PaletteManipulator::POSITION_APPEND)
-    ->addField('iso_store_id', 'isotope_legend', \Haste\Dca\PaletteManipulator::POSITION_APPEND)
-    ->applyToPalette('root', 'tl_page')
-;
+try {
+    \Haste\Dca\PaletteManipulator::create()
+        ->addLegend('isotope_legend', 'publish_legen', \Haste\Dca\PaletteManipulator::POSITION_BEFORE)
```

Comment:
```suggestion
        ->addLegend('isotope_legend', 'publish_legend', \Haste\Dca\PaletteManipulator::POSITION_BEFORE)
```
Or is the missing `d` deliberate?

Reply:
@aschempp? 😆